lib/px/block/pxblock.lua

```diff
@@ -5,8 +5,12 @@
 ----------------------------------------------
 
 local ngx_HTTP_FORBIDDEN = ngx.HTTP_FORBIDDEN
+local nginx_HTTP_TEMPORARY_REDIRECT = ngx.HTTP_TEMPORARY_REDIRECT
```

Comment:
Also need to update the README.MD file to reflect the minimum version is now v0.9.20 for the NGX Lua (HTTP_TEMPORARY_REDIRECT was added as a status code in this version) or add "ngx.HTTP_TEMPORARY_REDIRECT or 307" to keep the minimum required version at v0.9.11